src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -300,31 +300,34 @@ public function moduleAction(Request $request)
                     $response[$module]['status'] = $moduleManager->{$action}($module);
                 }
 
-                if ($response[$module]['status'] === null) {
-                    $response[$module]['status'] = false;
+                if ($response[$module]['status'] === true) {
                     $response[$module]['msg'] = $translator->trans(
-                        '%module% did not return a valid response on %action% action.',
+                        '%action% action on module %module% succeeded.',
                         array(
+                            '%action%' => ucfirst(str_replace('_', ' ', $action)),
                             '%module%' => $module,
-                            '%action%' => $action, ),
+                        ),
                         'Admin.Modules.Notification'
                     );
                 } elseif ($response[$module]['status'] === false) {
-                    $error = $moduleManager->getError($module);
+                    $error                    = $moduleManager->getError($module);
                     $response[$module]['msg'] = $translator->trans(
                         'Cannot %action% module %module%. %error_details%',
                         array(
-                            '%action%' => str_replace('_', ' ', $action),
-                            '%module%' => $module,
-                            '%error_details%' => $error, ),
+                            '%action%'        => str_replace('_', ' ', $action),
+                            '%module%'        => $module,
+                            '%error_details%' => $error,
+                        ),
                         'Admin.Modules.Notification'
                     );
                 } else {
-                    $response[$module]['msg'] = $translator->trans(
-                        '%action% action on module %module% succeeded.',
+                    $response[$module]['status'] = false;
+                    $response[$module]['msg']    = $translator->trans(
+                        '%module% did not return a valid response on %action% action.',
                         array(
-                            '%action%' => ucfirst(str_replace('_', ' ', $action)),
-                            '%module%' => $module, ),
+                            '%module%' => $module,
+                            '%action%' => $action,
```

Comment:
There is a mistake in the original code. Would you mind using `str_replace('_', ' ', $action)` here too?

Reply:
Yes, no problem.